content/token-program.md

```diff
@@ -393,6 +396,147 @@ async function buildBurnTransaction(
 }
 ```
 
+## Freeze Tokens
+Freezing token is the process of freezing token account so that token account owner can't transfer or burn tokens,by the freezeing authority of the token mint.
+
+To freeze tokens using the `spl-token` library, you use the `freezeAccount` function.
+
+```tsx
```

Comment:
```suggestion
```typescript
```

Reply:
Just replace all the `tsx` that aren't TSX with `typescript` please. I know that's not your code, but we may as well fix it now.

Likewise please ensure the code examples are formatted with 2 spaces per `CONTRIBUTING.md` / prettier. Thanks!